helix-term/src/ui/statusline.rs

```diff
@@ -254,6 +257,14 @@ where
     );
 }
 
+fn render_primary_selection_length<F>(context: &mut RenderContext, write: F)
+where
+    F: Fn(&mut RenderContext, String, Option<Style>) + Copy,
+{
+    let tot_sel = context.doc.selection(context.view.id).primary().len();
+    write(context, format!(" {} ", tot_sel), None);
```

Comment:
adding the word 'char' to give some more context and handling the plural form in a similar manner as the selection counter would be a nice addition I think.

```suggestion
    write(context, format!(" {} char{} ", tot_sel, if tot_sel == 1 { "" } else { "s" }), None);
```